tests/testMaskedImage.py

```diff
@@ -127,6 +127,23 @@ def testArrays(self):
         mimage2 = afwImage.makeMaskedImageFromArrays(image, mask, variance)
         self.assertEqual(type(mimage2), type(self.mimage))
 
+    def testProperties(self):
+        self.assertImagesEqual(self.mimage.image, self.mimage.getImage())
+        self.assertImagesEqual(self.mimage.mask, self.mimage.getMask())
```

Comment:
I suggest `assertMasksEqual` instead. It makes the code clearer and there may come a time when it compares the mask plane dicts.